internal/controller/discoverymgr/mnedc/server/server.go

```diff
@@ -61,6 +62,7 @@ var (
 	tunIns         tunmgr.Tun
 	networkUtilIns connectionutil.NetworkUtil
 	log            = logmgr.GetInstance()
+	serverVirtualIP net.IP
```

Comment:
I think we don't need `serverVirtualIP` if we use generateServerIP() to return virtual IP.

Reply:
Ok